templates/daemon/docs/__skillname__/README.md

```diff
@@ -4,7 +4,22 @@ This workspace template demonstrates a simple daemon skill skeleton that simply
 
 ## Building and Publishing the Skill
 
+### Using the cortex cli
```

Comment:
same comments as in templates/job/docs/__skillname__/README.md. except for the daemon to job comment as this is a daemon